### ISSUE TITLE
has this problem resolved yet?

### ISSUE THREAD

**Issue by energygreek** [jazevedo620/grpc-async-multicast#1]
i can reproduce this same problem, do you find the cause?

**Comment by jazeved0** [comment]
No, the problem has not been resolved. It is being tracked here: https://github.com/grpc/grpc/issues/22510, but the maintainers aren't seeming to prioritize/recognize it. The workaround is to set the `GRPC_POLL_STRATEGY` environment variable to `epoll1`. Doing so results in the expected behavior.

**Comment by gualopezb** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
forgeTOE, @tianhu288, @ZuulAndRibbon, @xzq8, @mattmidd, @bbjun7189, @mazgalesc, @k6c8e4, @DoCmAL1, @Tiger59, @Kenji85xo, @foriacus, @Tomburgs, @nonanubis, @vadim4004, @turbospaces, @ohxxx, @shiny, @florianloechle, @TAG-Epic
